benchmarks/src/main/java/org/questdb/OooCppBenchmark.java

```diff
@@ -0,0 +1,172 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2020 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.questdb;
+
+import io.questdb.griffin.SqlException;
+import io.questdb.std.Os;
+import io.questdb.std.Rnd;
+import io.questdb.std.Unsafe;
+import io.questdb.std.Vect;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.runner.RunnerException;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class OooCppBenchmark {
+    private static final long BUFFER_MAX_SIZE = 256 * 1024 * 1024L;
+    private static long buffer;
+    private static final long MB = 1024 * 1024L;
+
+    public static void main(String[] args) throws RunnerException, SqlException, IOException {
+
+        Os.init();
+        System.out.println("MemorySuffle");
+        testSetMemoryShuffleToCsv();
+
+//        System.out.println("SetMemoryVanilla");
```

Comment:
remove?

Reply:
separated to another class, can be useful for further optimisations